packages/yoroi-extension/app/components/topbar/WalletListDialog.js

```diff
@@ -268,4 +271,125 @@ export default class WalletListDialog extends Component<Props, State> {
       </Dialog>
     );
   }
+
+  renderWalletsTotal(): Node {
+    const {
+      unitOfAccountSetting,
+      cardanoWallets,
+      ergoWallets,
+      shouldHideBalance,
+      getCurrentPrice,
+    } = this.props;
+    if (unitOfAccountSetting.enabled) {
+      let totalFiat;
+      const adaFiat = this.sumWallets(cardanoWallets).fiat;
+      const ergFiat = this.sumWallets(ergoWallets).fiat;
+      if (adaFiat == null || ergFiat == null) {
+        totalFiat = null;
+      } else {
+        totalFiat = adaFiat.plus(ergFiat);
+      }
+      const { currency } = unitOfAccountSetting;
+      return (
+        <FiatDisplay
+          shouldHideBalance={shouldHideBalance}
+          amount={totalFiat}
```

Comment:
The `<FiatDisplay />` will display a spinner if `totalFiat` is `null`  but if the backend is not working or we don't have the fiat amount the spinner will be there all the time 

<img width="250" height="250" alt="Screen Shot 2022-06-03 at 11 07 09 AM" src="https://user-images.githubusercontent.com/72753578/171825220-54d70e4e-4331-4f54-82e8-9c5761e76f05.png">
 
**Please don't display the *spinner* when the fiat amount is not available** like backend issue or something

Reply:
changed in new PR